client/hawc_client/epiv2.py

```diff
@@ -0,0 +1,538 @@
+        """
+        url = f"{self.session.root_url}/epidemiology/api/adjustment-factor/{adjustment_factor_id}/"
+        return self.session.delete(url)
+
+    def get_data_extraction(self, data_extraction_id: int) -> dict:
+        """
+        Retrieves data for a single data extraction
+
+        Args:
+            data_extraction_id (int): data extraction ID
+
+        Returns:
+            dict: data extraction data
+        """
+        url = f"{self.session.root_url}/epidemiology/api/data-extraction/{data_extraction_id}/"
+        return self.session.get(url).json()
+
+    def create_data_extraction(self, data: dict) -> dict:
+        """
+        Create a new data extraction
+
+        Args:
+            data (dict): fields to create on the data extraction. Keys:
+                design (int): id of the associated design
+                outcome_id (int): id of the associated outcome
+                exposure_level_id (int): id of the associated exposure level
+                factors_id (int): id of the associated adjustment factors
+                sub_population (str): subgroup within the study population (free-text)
+                outcome_measurement_timing (str): age or other timing, "cross-sectional", etc. (free-text)
+                effect_estimate_type (str): type of effect estimate (predefined choices: 'Odds Ratio'/'OR', 'Absolute Risk %'/'AR', etc.)
+                effect_estimate (float): value of effect estimate
+                ci_lcl (float): lower bound
+                ci_ucl (float): upper bound
+                ci_type (str): lower/upper bound type (predefined choices: 'P90', '5th/95h percentile', etc.)
+                units (str): bound units (free-text)
+                variance_type (str): type of variance estimate (predefined choices: 'SD', 'SE', 'NA', etc.)
+                variance (float): value of variance estimate
+                n (int): n
+                p_value (str): p-value (free-text)
+                significant (str): was the data extraction statistically significant? (predefined choices: 'NR', 'Yes', etc.)
+                group (str): results group for linking sets of results (free-text)
+                exposure_rank (int): value for ordering linked sets of results
+                exposure_transform (str): exposure transform (predefined choices: 'log10', 'squared', 'other', etc.)
+                outcome_transform (str): outcome transform (predefined choices: 'log10', 'squared', 'other', etc.)
+                confidence (str): overall confidence rating for the endpoint being extracted (free-text)
+                data_location (str): e.g. table number (free-text)
+                effect_description (str): description of the effect estimate with units, including comparison being made. (free-text)
+                statistical_method (str): brief description of the statistical analysis method (free-text)
+                comments (str): general comments (free-text)
```

Comment:
add adjustment_factor to client; see https://github.com/shapiromatron/hawc/pull/902

Reply:
@shapiromatron you mean "adverse_direction" right? (assuming "adjustment_factor" was a typo)

Reply:
yes! my bad, thank you for translating my error!